configs/opencv-ovms/demos/classification/python/entrypoint.sh

```diff
@@ -5,5 +5,5 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 python3 classification/python/classification_demo.py -m localhost:"$GRPC_PORT"/models/"$CLASSIFICATION_MODEL_NAME" \
-	--label classification/python/labels/"$CLASSIFICATION_LABEL_FILE" -i $inputsrc \
+	--label classification/python/labels/"$CLASSIFICATION_LABEL_FILE" -i $INPUTSRC \
```

Comment:
I have an open PR that may be affected by this change. Let's just make sure we sync up. 
I'm setting this variable as env var on the run_classification.sh script

Reply:
Same for all changes below